doc/api/dgram.md

```diff
@@ -95,6 +95,17 @@ Tells the kernel to join a multicast group at the given `multicastAddress` and
 one interface and will add membership to it. To add membership to every
 available interface, call `addMembership` multiple times, once per interface.
 
+When the socket is created, each worker gets a copy and they refer the
+same socket so you need to manage to call `.addMembership` only once.
```

Comment:
Please avoid using the informal pronoun `you` in the docs :-)

Reply:
Sure! I will update it.